### DISCUSSION TITLE
dual grid question

### DISCUSSION THREAD

**Discussion by fenoteryn** [bosskmk/pluto_grid#166] [Q&A]
dual grid을 사용하려고 하는데 A 그리드와 B 그리드의 방향이 가로가 아닌 세로로 바꿀 수 있는 방법이 있나요?

**Comment by bosskmk** [comment]
기능을 추가 해야 합니다. 그게 아니면 기본 PlutoGrid 로 직접 구현하실 수 있습니다.
PlutoDualGrid 의 소스코드를 확인해보십시요.
좌우로 되어있는 것을 위아래로 Layout 을 구현하시면 됩니다.

**Reply by fenoteryn** [reply]
좌우가 아닌 위아래로 구현할 수 있는 설정이 있나요? 듀얼그리드 소스코드를 확인해도 어떤 코드가 위아래로 Layout을 바꿔 주는 건지 모르겠습니다...

**Reply by bosskmk** [reply]
앞서 말씀드린 것처럼 기능을 추가 해야 합니다.
좌우로 되어있는 소스코드를 참고하셔서 위아래로 구현하셔야 합니다.
